src/applications/personalization/profile-2/components/ProfileInfoTable.jsx

```diff
@@ -1,37 +1,109 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const ProfileInfoTable = ({ data, dataTransformer, fieldName, title }) => (
-  <table className="profile-info-table" data-field-name={fieldName}>
-    <thead>
-      <tr>
-        <th>
-          <h3>{title}</h3>
-        </th>
-      </tr>
-    </thead>
-    <tbody>
+/**
+ * Helper that prefixes class names with an optional responsive prefix and the
+ * `vads-u-` utility class prefix.
+ *
+ * @param {string[]} classes - Array of classes to prefix with `vads-u-`
+ * @param {string} screenSize - Optional screen size
+ * @returns {string[]} The input `classes` array with the correct prefixes
+ * applied
+ *
+ * Example: `prefixUtilityClasses(['my-class'], 'medium')` returns
+ * ['medium-screen:vads-u-my-class']
+ */
+const prefixUtilityClasses = (classes, screenSize = '') => {
```

Comment:
I didn't like having `vads-u-` repeated in every array of class names so I made this helper to prefix them for me.

As an aside, I really hated throwing all these strings around in JS. I constantly typed them wrong and wouldn't know until after a rebuild and load in the browser. Would be great if these design system class names lived in constants we could use. They could be imported from the formation NPM package and used in our React components.